pkg/kube/inject/testdata/inject/cronjob.yaml.injected

```diff
@@ -10,14 +10,13 @@ spec:
         prometheus.io/path: /stats/prometheus
         prometheus.io/port: "15020"
         prometheus.io/scrape: "true"
-        sidecar.istio.io/interceptionMode: REDIRECT
         sidecar.istio.io/status: '{"version":"","initContainers":["istio-init"],"containers":["istio-proxy"],"volumes":["istio-envoy","istio-data","istio-podinfo","istio-token","istiod-ca-cert"],"imagePullSecrets":null}'
-        traffic.sidecar.istio.io/excludeInboundPorts: "15020"
```

Comment:
Why are the other annotations not coming through?

Reply:
See comment on https://github.com/istio/istio/pull/26596#discussion_r472327509. tl;dr existing bug in webhook that impacts CNI. This has *never* worked since it was introduced - it was always istioctl kube-inject only. Since the webhook is the primary code path, its safer to keep its logic. Stewart will followup with ensuring CNI gets this functionality working, which may or may not involve adding these annotations back